Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI_description.md

```diff
@@ -1,8 +1,10 @@
+Provide the API KEY by following the instructions:  
 <~XSIAM>
-Creating API keys is done in the Cortex XSIAM interface, under Settings -> Configurations -> API Keys
+Creating API keys is done in the Cortex XSIAM interface, under Settings -> Configurations -> API Keys.  
```

Comment:
You can use similar description from XDR IR integration

### Generate an API Key and Auth ID
1. In your Cortex XSIAM platform, go to **Settings**.
2. Click the **+New Key** button in the top right corner.
3. Generate a key of type **Advanced**.
4. Copy and paste the key.
5. From the ID column, copy the Key ID.

### URL
1. In your Cortex XSIAM platform, go to **Settings**.
2. Click the **Copy URL** button in the top right corner.